src/Monolog/Formatter/JsonFormatter.php

```diff
@@ -100,6 +103,22 @@ public function includeStacktraces($include = true)
         $this->includeStacktraces = $include;
     }
 
+    /**
+     * @return int
+     */
+    public function getMaxDepth()
+    {
+        return $this->maxDepth;
+    }
+
+    /**
+     * @param int $maxDepth
+     */
+    public function setMaxDepth($maxDepth)
+    {
+        $this->maxDepth = $maxDepth;
+    }
```

Comment:
This here and the property declaration above + constructor assignment should probably be reused from NormalizerFormatter as this class extends it.